src/iccpd/src/mclagdctl/mclagdctl.h

```diff
@@ -142,6 +142,7 @@ struct mclagd_state
     int session_timeout;
     int keepalive_time;
     char loglevel[MCLAGDCTL_PARA1_LEN];
+    uint8_t peer_link_state;
```

Comment:
Is this declaration correct?  The usage seems to try to use it as a string but this looks like a single char (uint8) so wouldn't this be a problem??